bittensor/wallet.py

```diff
@@ -140,6 +142,10 @@ def add_args(cls, parser: argparse.ArgumentParser, prefix: str = None):
         """
         prefix_str = "" if prefix == None else prefix + "."
         try:
+            # TODO: Where should the default values be stored? Currently, they are scattered all across the codebase.
```

Comment:
So I'm fine if the *defaults* are stored here and then they read the config or default. I'd rip out any references to looking up environment variables outside of the config class. It looks like we will have to modify this class in order to get the wallet lookup to use the profiles.